src/stream/src/executor/aggregation/agg_impl/foldable.rs

```diff
@@ -429,9 +429,17 @@ impl_fold_agg! { F64Array, Float64, F32Array }
 
 #[cfg(test)]
 mod tests {
-    use risingwave_common::array::I64Array;
+    extern crate test;
+
+    use itertools::Itertools;
+    use rand::seq::SliceRandom;
+    use rand::Rng;
+    use risingwave_common::array::stream_chunk::Op;
+    use risingwave_common::array::{Array, ArrayBuilder, I64Array, I64ArrayBuilder};
+    use risingwave_common::buffer::{Bitmap, BitmapBuilder};
     use risingwave_common::types::F64;
     use risingwave_common::{array, array_nonnull};
+    use test::Bencher;
```

Comment:
I think we should use `Criterion` for benchmarking, it is more stable than `bencher`.

Reply:
that being said, if it's too much trouble to migrate, and bencher serves our purposes here (results are accurate and consistent), feel free to ignore this comment.

Reply:
I've tried using Criterion for benchmarking, but the excellent encapsulation of the test object failed my attempt :(

Reply:
FYI: https://github.com/bheisler/criterion.rs/issues/158